### PR TITLE
Move package to com.localytics and general cleanup

### PR DESCRIPTION
**Pull request by bdarfler**

### CODE REVIEW

**Comment by bdarfler** [comment]
@jsolmon when you get back give this a look

File: build.sbt [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
 
-bintrayOrganization in bintray := None
+// Error on conflicting dependencies
+// http://www.scala-sbt.org/0.13/docs/Library-Management.html#Conflict+Management
+conflictManager := ConflictManager.strict
 
-licenses += ("MIT", url("http://opensource.org/licenses/MIT"))
+// Error on circular dependencies
+// http://www.scala-sbt.org/0.13/docs/sbt-0.13-Tech-Previews.html#Circular+dependency
+updateOptions := updateOptions.value.withCircularDependencyLevel(CircularDependencyLevel.Error)
```
Comment (jsolmon) [inline build.sbt]: I like the checks for circular/conflicting dependencies!

**Comment by jsolmon** [comment]
This looks good so far @bdarfler

**Comment by bdarfler** [comment]
@jsolmon I'm thinking we pull this PR in and then I'll start another to get it to give with our `fun:test` and `it:test` setup. :shipit: ?

**Comment by jsolmon** [comment]
:shipit:

**Pull request merged**